includes/discovery/functions.inc.php

```diff
@@ -953,3 +953,33 @@ function ignore_storage($descr)
 
     return $deny;
 }
+
+/**
+ * @param $types
+ * @param $device
+ * @param array $pre_cache
+ */
+function sensors($types, $device, $pre_cache = array())
+{
+    global $config;
+    foreach ((array)$types as $type) {
+        echo ucfirst($type) . ': ';
+
+        $dir = $config['install_dir'] . '/includes/discovery/sensors/' . $type .'/';
+
+        if (is_file($dir . $device['os_group'] . '.inc.php')) {
+            include_once $dir . $device['os_group'] . '.inc.php';
+        }
+        if (is_file($dir . $device['os'] . '.inc.php')) {
+            include_once $dir . $device['os'] . '.inc.php';
+        }
+        if (isset($config['modules_compat']['rfc1628'][$device['os']]) && $config['modules_compat']['rfc1628'][$device['os']]) {
+            if (is_file($dir  . '/rfc1628.inc.php')) {
```

Comment:
Why can we ignore the checks? the rfc1628 aren't named after groups or os so need calling.

Not sure what you mean by avoid the is_file checks, do you mean just call include_once and not worry about no file existing?